src/greyVest/Dynamic.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import { observer } from 'mobx-react'
+
+let Dynamic = ({ component: C = null, defaultProps, ...props }) =>
+  C && (
+    <C
+      {..._.flow(
+        _.pickBy(_.identity),
```

Comment:
`F.compactObject` does exactly this: https://github.com/smartprocure/futil-js/blob/master/src/object.js#L27